rpcd/playbooks/roles/rpc_support/tasks/holland_config.yml

```diff
@@ -31,6 +31,7 @@
       password: "{{ rpc_support_holland_password }}"
       priv: "*.*:ALL"
       state: present
+  when: inventory_hostname == groups['galera'][0]
```

Comment:
Ansible has created 'run_once' which is considered to be cleaner than using when.

http://docs.ansible.com/ansible/playbooks_delegation.html#run-once